com.termius.termius.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.termius.termius
+
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: "18.08"
+
+command: termius-app
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=host
+  # Allow access to keyring
+  - --talk-name=org.freedesktop.secrets
+  # SSH
+  - --socket=ssh-auth
+  - --own-name=org.KDE.*
+
+modules:
+
+  - name: termius
+    buildsystem: simple
+    build-commands:
+        - install -Dm755 termius-app.sh /app/bin/termius-app
+        - install -Dm755 apply_extra.sh /app/bin/apply_extra
+        - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata/
+        - install -Dm755 /usr/bin/desktop-file-edit -t /app/bin/
```

Comment:
Why you do install those into /app/bin?

Reply:
Because those are used in `apply_extra`.

Reply:
Well, you can use them without copying them over to /app prefix. /usr/bin is on $PATH too AFAIK.

Reply:
desktop-file-utils are present in the SDK, but not in the runtime. Same as `ar`.